prboom2/src/dsda/features.h

```diff
@@ -25,7 +25,7 @@ typedef enum {
   uf_quickstartcache,
   uf_100k,
   uf_console,
-  // 7
+  uf_levelbrightness,
```

Comment:
This needs to be at the end of the enum, really cannot be here.
Another PR has just been merged that adds features to the end as well, so this needs to be at 67